sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c

```diff
@@ -184,10 +265,18 @@ void guidance_indi_calcG(struct FloatMat33 *Gmat) {
   RMAT_ELMT(*Gmat, 2, 2) = cphi*ctheta;
 }
 
-void stabilization_attitude_set_setpoint_rp_quat_f(bool in_flight, int32_t heading)
+/**
+ * @param indi_rp_cmd roll/pitch command from indi guidance [rad]
+ * @param in_flight in flight boolean
+ * @param heading the desired heading [rad]
```

Comment:
nitpick: this is radians in BFP with INT32_ANGLE_FRAC, whereas the indi_rp_cmd is really in radians...